keyboards/kibou/suisei/keymaps/default/keymap.c

```diff
@@ -0,0 +1,21 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
this looks like it matches `LAYOUT_65_ansi_split_bs` community layout.  

If it does, support for community layouts would be ideal.

Reply:
Yes, my code does match the 'LAYOUT_65_ansi_split_bs' community layout. I'll do my best to support this layout to make sure my code works correctly on keyboards that use this layout.

Reply:
Thanks!